source/PlayerInfo.cpp

```diff
@@ -2820,6 +2822,12 @@ void PlayerInfo::Save(const string &path) const
 	DataWriter out(path);
 
 
+	// Current game version:
+	out.Write("version", Version::GetVersion());
+	const auto commitId = Version::GetCommit();
+	if(!commitId.empty())
+		out.Write("commit", commitId);
```

Comment:
```suggestion
		out.Write("version commit", commitId);
```

@petervdmeer thoughts?

Reply:
The keywords `version` and `version commit` are okay to me.

We could go for the following structure with child keywords if we want to group it nicely: 
```
version
	number 0.9.15.alpha
	commit abc0123456789...
```
But I'm already okay with the keywords as you suggested.

Reply:
One afterthought, if we are also going to store the version-numbers of the data-files and [version-numbers of installed plugins](https://github.com/endless-sky/endless-sky/pull/7084), then we might be more explicit that this is the engine version;
```
"engine version"
	number 0.9.15.alpha
	commit abc0123456789...
```

Reply:
```
version 0.9.15
    commit abc123
```
Not sure I like `number` there.